packages/react/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { Masthead } from '../Masthead';
+import { Footer } from '../Footer';
+import { init } from '../../global';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * DotcomShell component
+ *
+ * @typedef {object} navigation Object containing navigation elements
+ * @param {string} User content
+ * @returns {*} DotcomShell component
+ */
+const Layout = ({ navigation, footerType, children, ...mastheadProps }) => {
+  useEffect(() => {
+    init();
+  }, []);
+
+  return (
+    <>
+      <Masthead navigation={navigation} {...mastheadProps} />
```

Comment:
@andysherman2121 this doesn't look right, you shouldn't have anything like masthead or footer in here